experiments/process_sandbox/platform/syscall_context_freebsd.h

```diff
@@ -0,0 +1,153 @@
+// Copyright Microsoft and Project Verona Contributors.
+// SPDX-License-Identifier: MIT
+#ifdef __FreeBSD__
+#  include <sys/syscall.h>
+#  include <ucontext.h>
+
+namespace sandbox::platform
+{
+#  ifdef __x86_64__
+  /**
+   * FreeBSD system call information.
+   *
+   * Note: This depends on FreeBSD D29185 landing.
+   */
+  class SyscallFrameFreeBSDX8664
```

Comment:
Nit: I'd call this and the Linux classes `...X64` instead of `X8646`.

Reply:
Good question, don't know. My dyslexia doesn't let me know that this is about `x86_64` unless I stare at it for a while. Maybe it's not a problem for others. Not an important point, though. Feel free to ignore.